### PR TITLE
Fixed not being able to construct NTNet Relays

### PR DESCRIPTION
**Pull request by Shadow-Quill**
This is untested, so bewarned.

### CODE REVIEW

**Comment by Repede** [comment]
I'd ask to test it locally and maybe make that PR upstream (Polaris I assume) or at least add //YW changes to the lines that were changed at a minimum.

File: code/game/objects/items/weapons/circuitboards/machinery/research.dm [hunk 1]
```diff
@@ -89,8 +89,8 @@ obj/item/weapon/circuitboard/rdserver/attackby(obj/item/I as obj, mob/user as mo
 
 obj/item/weapon/circuitboard/ntnet_relay
 	name = "Circuit board (NTNet Quantum Relay)"
-	build_path = "/obj/machinery/ntnet_relay"
-	board_type = "machine"
+	build_path = /obj/machinery/ntnet_relay
+	board_type = new /datum/frame/frame_types/machine
```
Comment (Repede) [inline code/game/objects/items/weapons/circuitboards/machinery/research.dm]: Add //YW changes in front

File: code/game/objects/items/weapons/circuitboards/machinery/research.dm [hunk 2]
```diff
@@ -89,8 +89,8 @@ obj/item/weapon/circuitboard/rdserver/attackby(obj/item/I as obj, mob/user as mo
...
 	origin_tech = list(TECH_DATA = 4)
 	req_components = list(
-							"/obj/item/stack/cable_coil" = 15)
+							/obj/item/stack/cable_coil = 15)
```
Comment (Repede) [inline code/game/objects/items/weapons/circuitboards/machinery/research.dm]: Add //YW changes in front

**Comment by Shadow-Quill** [comment]
Now tested, works as it should.

https://github.com/PolarisSS13/Polaris/pull/6570 for upstream pr

**Pull request merged**